cmd/cmd.go

```diff
@@ -30,7 +31,12 @@ func NewRootCmd(
 					dbConnector,
 					dbMigrationTool,
 					securityPolicy,
+					eventDispatcher,
 				)
+
+				if err := eventDispatcher.Close(); err != nil {
```

Comment:
Reading this file makes me wonder whether we should actually initialize the eventDispatcher here :stuck_out_tongue:. What are your thoughts?